releases/1.8/edge/kubeflow/bundle.yaml

```diff
@@ -0,0 +1,273 @@
+  kfp-metadata-writer:
+    charm: kfp-metadata-writer
+    channel: 2.0/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-persistence:
+    charm: kfp-persistence
+    channel: 2.0/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-profile-controller:
+    charm: kfp-profile-controller
+    channel: 2.0/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-schedwf:
+    charm: kfp-schedwf
+    channel: 2.0/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-ui:
+    charm: kfp-ui
+    channel: 2.0/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-viewer:
+    charm: kfp-viewer
+    channel: 2.0/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  kfp-viz:
+    charm: kfp-viz
+    channel: 2.0/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kfp-operators
+  knative-eventing:
+    charm: knative-eventing
+    channel: 1.10/edge
+    scale: 1
+    trust: true
+    options:
+      namespace: knative-eventing
+    _github_repo_name: knative-operators
+  knative-operator:
+    charm: knative-operator
+    channel: 1.10/edge
+    scale: 1
+    trust: true
+    _github_repo_name: knative-operators
+  knative-serving:
+    charm: knative-serving
+    channel: 1.10/edge
+    scale: 1
+    trust: true
+    options:
+      namespace: knative-serving
+      istio.gateway.namespace: kubeflow
+      istio.gateway.name: kubeflow-gateway
+    _github_repo_name: knative-operators
+  kserve-controller:
+    charm: kserve-controller
+    channel: 1.8/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kserve-operators
+  kubeflow-dashboard:
+    charm: kubeflow-dashboard
+    channel: 1.8/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kubeflow-dashboard-operator
+  kubeflow-profiles:
+    charm: kubeflow-profiles
+    channel: 1.8/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kubeflow-profiles-operator
+  kubeflow-roles:
+    charm: kubeflow-roles
+    channel: 1.8/edge
+    scale: 1
+    trust: true
+    _github_repo_name: kubeflow-roles-operator
+  kubeflow-volumes:
+    charm: kubeflow-volumes
+    channel: 1.8/edge
+    scale: 1
+    _github_repo_name: kubeflow-volumes-operator
+  metacontroller-operator:
+    charm: metacontroller-operator
+    channel: 3.0/edge
+    scale: 1
+    trust: true
+    _github_repo_name: metacontroller-operator
+  mlmd:
+    charm: mlmd
+    channel: 1.14/edge
+    scale: 1
+    _github_repo_name: mlmd-operator
+  minio:
+    charm: minio
+    channel: ckf-1.8/edge
+    scale: 1
+    _github_repo_name: minio-operator
+  oidc-gatekeeper:
+    charm: oidc-gatekeeper
+    channel: ckf-1.8/edge
+    scale: 1
+    trust: true
+    _github_repo_name: oidc-gatekeeper-operator
+  pvcviewer-operator:
+    charm: pvcviewer-operator
+    channel: 1.8/edge
+    scale: 1
+    trust: true
+    series: focal
```

Comment:
is series required here?

Reply:
yes, please see [my other comment](https://github.com/canonical/bundle-kubeflow/pull/733#discussion_r1400248494)